faris-wireframe-userstory/public/main.css

```diff
@@ -0,0 +1,143 @@
+body,html {
+    margin: 0;
+    padding: 0;
+    font-family: Verdana;
+  }
+p {
+    padding-right: 70px;
+    word-break: break-word;
+}
+/* Flex box css */
+.container {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+    height: 400px;
+}
+.form-container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+    margin-bottom: 50px;
+}
+.footer-container {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: nowrap;
+    margin-left: 100px;
+    margin-right: 100px;
+}
+.item {
+    width: 100%;
+}
+#nav {
+    z-index: 2;
+    margin-top: 80px;
+}
+footer button {
+    width: 100px;
+    height:30px;
+    margin-left: 20px;
+    background:#cccccc;
+    border: none;
+}
+/*  */
+.sub-title {
+    color:black;
+    text-align: center;
+    margin-bottom: 50px;
+}
+.grey-box {
+    width: 90%;
+    height: 90%;
+    background:#cccccc;
+}
+
+header {
+    height: 400px;
+    background: #464646;
+    margin-bottom:50px; 
+    width: 100%
+}
+
+footer {
+    background: #464646;
+    height: 100px;
+    font-size: 18px;
+    vertical-align: middle;
+    line-height: 100px;
+    width: 100% 
+}
+
+/* form */
+form {
```

Comment:
The `<form></form>` element can be used on multiple places across the website. So it won't be smart to do it like this. Use classes instead.